scripts/stats-dashboard/plot-mp-coverage-ratio.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+import argparse
+import matplotlib.pyplot as plt
+import os, re
+import pandas as pd
+import pandas.api.types as pdtypes
+
+
+
+
+here = os.path.dirname(__file__)
+
+
+def plot(df):
+    versions = df.columns
+    versions = sorted(set(versions), key=lambda v: list(map(int, v[1:].split('.'))), reverse=True)
+    versions = versions[:4]
+    df = df[versions]
+    p, a = plt.subplots()
+    a.plot(df)
+    a.axhline(y=1, color='green', linestyle='--', linewidth=1, label='_nolegend_')
+    a.set_title("Ratio: members of parliament to seats")
+    a.legend(versions, loc ="upper left")
+    p.savefig(f"{here}/figures/mp-coverage/mp-coverage-ratio.png")
+
+
+
+
+def main(args):
+
+    skip = [
+        'prot-1909----reg-01.xml',
+        'prot-1909----reg-02.xml',
+        'prot-197677--.xml',
+        'prot-197778--.xml',
+        ] # these were "test" protocols & break code :|
```

Comment:
Nice! Maybe move out these protocols as a separate functions since this is skipped in multiple places and add an explainaition. Should these records be removed?